lib/Output/JSON_renderer.php

```diff
@@ -185,12 +188,14 @@ public static function addCompleteArray($in, $request)
                            "permalinkAlt" => localize("permalink.alternate"),
                            "publisherName" => localize("publisher.name"),
                            "pubdateTitle" => localize("pubdate.title"),
-                           "languagesTitle" => localize("language.title"),
+                           "languagesTitle" => localize("languages.title"),
+                           "languageTitle" => localize("language.title"),
                            "contentTitle" => localize("content.summary"),
                            "filterClearAll" => localize("filter.clearall"),
                            "sortorderAsc" => localize("search.sortorder.asc"),
                            "sortorderDesc" => localize("search.sortorder.desc"),
                            "customizeEmail" => localize("customize.email")],
+                           "ratingsTitle" => localize("ratings.title"),
```

Comment:
This looks like the ] from the previous line should be moved too, no?

Reply:
Thanks for catching that. I've fixed it and committed.